datalad_service/handlers/description.py

```diff
@@ -0,0 +1,39 @@
+import falcon
+import os
+
+from datalad_service.tasks.description import update_description
+
+
+class DescriptionResource(object):
+    def __init__(self, store):
+        self.store = store
+
+    @property
+    def annex_path(self):
+        return self.store.annex_path
+
+    def on_post(self, req, resp, dataset):
+        """
+        Commmit a description change.
+        Returns update dataset_description
+        """
+        if dataset:
+            try:
+                description_fields = req.media.get('description_fields')
+                if not any(description_fields):
+                    resp.media = {
+                        'error': 'Missing description field updates.'
+                    }
+                    resp.status = falcon.HTTP_UNPROCESSABLE_ENTITY
+                dataset_description = update_description(self.store.annex_path, dataset, None, description_fields)
```

Comment:
This does not call the task via Celery so this will occur unordered. Need to use .apply_async or .s() and call to use the task queue. Also this call needs to occur on the correct queue (see other tasks for an example).